src/sessionTypes/auth0WebAuth.js

```diff
@@ -303,7 +394,7 @@ class Auth0WebAuth {
     return {
       accessToken: localStorage.getItem('access_token'),
       apiToken: localStorage.getItem('api_token'),
-      expiresAt: localStorage.getItem('expires_at')
+      expiresAt: JSON.parse(localStorage.getItem('expires_at'))
```

Comment:
What's the purpose of moving to `JSON.parse` and `JSON.stringify` when fetching, setting the expiry?

Reply:
When looking at the Auth0 docs surrounding Silent Authentication, it had that transition happening. When investigating, I was reminded that local storage stores everything as strings, so it was getting coerced from a number to a string when storing it, and then back from string to number when comparing it/adding it/subtracting it elsewhere. I figured it was good to follow their lead and make it a little more explicit what was happening. Thoughts?

Reply:
AH, yes... I like this better. Definitely keep as-is!!

Reply:
Sorry, that's super confusing, I mean keep the changes that you made :)